src/app/containers/SwapFormContainer/index.tsx

```diff
@@ -210,15 +218,13 @@ export function SwapFormContainer() {
         <SendTxProgress {...tx} displayAbsolute={false} />
 
         <div className="swap-btn-container">
-          <div className="swap-btn-helper">
-            <span>Maximum Received: 0.02931216</span>
-            <span>
-              <img
-                src={settingIcon}
-                alt="settings"
-                onClick={() => setDialogOpen(true)}
-              />
-            </span>
+          <div className="swap-btn-helper tw-flex tw-items-center tw-justify-center">
+            <span>Minimum Received: {weiToNumberFormat(minReturn, 8)}</span>
+            <img
+              src={settingIcon}
+              alt="settings"
+              onClick={() => setDialogOpen(true)}
+            />
           </div>
           <button
```

Comment:
Change button with component that was used in BuySovPage form, to make sure it has all states (hover, disabled, etc)

If user is not engaged wallet, button should say "Engage Wallet" - opens engage wallet dialog when clicked